ecpluginbuilder.pl

```diff
@@ -14,7 +15,7 @@
 
 my $epb="../ecpluginbuilder";
 
-my $pluginVersion = "0.9.1";
+my $pluginVersion = "0.9.2";

```

Comment:
I guess this will also need to be bumped to v 1.0.0.

Reply:
ecpluginbuilder.pl automatically update plugin.xml
it's not necessary for my own build but that way I'm sure the official build won't be out of version